pkgs/build-support/bintools-wrapper/default.nix

```diff
@@ -131,6 +131,8 @@ let
       ""
     else if targetPlatform.libc == "musl" then
       "${sharedLibraryLoader}/lib/ld-musl-*"
+    else if targetPlatform.libc == "mlibc" then
+      "${sharedLibraryLoader}/lib/ld.so"
```

Comment:
This path hasn't been "standardised" upstream, and might change (probably to follow musl paradigms). Would that be a breaking change in the future?